book/src/configuration.md

```diff
@@ -1,5 +1,7 @@
 # Configuring Clippy
 
+_NOTE: The configuration file is unstable and may be deprecated in the future._
```

Comment:
You can crate blocks, by writing this as

```suggestion
> **Note:** The configuration file is unstable and may be deprecated in the future.
```

At least this is what the Rust reference does: https://github.com/rust-lang/reference/blob/master/src/introduction.md#conventions